sn_client/src/files/api.rs

```diff
@@ -41,7 +41,11 @@ pub struct FilesApi {
     wallet_dir: PathBuf,
 }
 
-type ChunkFileResult = Result<(XorName, u64, Vec<(XorName, PathBuf)>)>;
+/// Result of the resulting data address, the DataMap Chunk, filesize and the paths of the chunks to be stored.
```

Comment:
what is that `data address` ?  the chunk_name of the `DataMap chunk` , or anything else ?